lupa/tests/test.py

```diff
@@ -754,6 +754,28 @@ def test():
             raise ValueError("huhu")
         self.assertRaises(ValueError, function, test)
 
+    def test_reraise_pcall(self):
+        function = self.lua.eval(
+            'function(p) local r, err = pcall(p); return r, err end'
+        )
+        self.assertEqual(
+            function(lambda: 5/0),
+            (False, "error during Python call")
+        )
+
+    def test_lua_error_after_intercepted_python_exception(self):
```

Comment:
says ok here: https://travis-ci.org/scoder/lupa/jobs/127188463
in which exactly configuration?